cterasdk/edge/config.py

```diff
@@ -18,6 +18,7 @@ def set_location(self, location):
         """
         Set the location of the gateway
 
+        :param str hostname: New location to set
```

Comment:
typo in the parameter name. Should be `location`

Reply:
fixed